File: PARQme/src/com/objects/ParkInstanceObject.java
package com.objects;

public class ParkInstanceObject {
	private long endTime;
	private String parkingReferenceNumber;
	private ParkSync sync;
	
	
	public ParkInstanceObject(long endTime, String parkingReferenceNumber,
			ParkSync sync) {
		super();
		this.endTime = endTime;
		this.parkingReferenceNumber = parkingReferenceNumber;
		this.sync = sync;
	}
	public ParkSync getSync() {
		return sync;
	}
	public void setSync(ParkSync sync) {
		this.sync = sync;
	}
	public long getEndTime() {
		return endTime;
	}
	public void setEndTime(long endTime) {
		this.endTime = endTime;
	}
	public String getParkingReferenceNumber() {
		return parkingReferenceNumber;
	}
	public void setParkingReferenceNumber(String parkingReferenceNumber) {
		this.parkingReferenceNumber = parkingReferenceNumber;
	}
}

File: parq-ios/Parq/ParkInstanceObject.m
//
//  ParkInstanceObject.m
//  Parq
//
//  Created by Michael Xia on 1/23/12.
//  Copyright (c) 2012 Massachusetts Institute of Technology. All rights reserved.
//

#import "ParkInstanceObject.h"

@implementation ParkInstanceObject

@synthesize parkingReferenceNumber;
@synthesize endTime;

-(void) setSync:(ParkSync *)syncIn{
    sync = syncIn;
}
-(ParkInstanceObject*) initWithNum:(NSString*)parkRefNum End:(NSNumber*)endTimeIn Sync:(ParkSync*)syncObj{
    parkingReferenceNumber = parkRefNum;
    endTime = endTimeIn;
    sync = syncObj;
    return self;
}
@end
